service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/generator/C43aSpecialGuardianshipOrderDocumentParameterGenerator.java

```diff
@@ -44,9 +44,9 @@ public class C43aSpecialGuardianshipOrderDocumentParameterGenerator implements D
         + "to remove the child from the United Kingdom without leave of the Court.\n"
         + "";
     private static String NOTICE_HEADER = "Notice \n";
-    private static String NOTICE_MESSAGE = "Any person with parental responsibility for a child may "
+    protected static String NOTICE_MESSAGE = "Any person with parental responsibility for a child may "
         + "obtain advice on what can be done to prevent the issue of a passport to the child. They should write "
-        + "to The United Kingdom Passport Agency, Globe House, 89 Eccleston Square, LONDON, SW1V 1PN.";
+        + "to Glasgow CPST, HMPO Glasgow, 96 Milton Street, Glasgow, G4 0BT or email Glasgowcaveats@hmpo.gov.uk.";
```

Comment:
once in config, the address can be reused here

Reply:
moved to config